app/assets/javascripts/actions/recent_edits_actions.js

```diff
@@ -0,0 +1,16 @@
+import { RECEIVE_RECENT_EDITS, API_FAIL } from "../constants";
+import API from "../utils/api.js";
+
+export const fetchRecentEdits = (opts = {}) => dispatch => {
+  return (
+    API.fetchRecentEdits(opts)
```

Comment:
Since this only gets called from here, I think we should move the ajax function from API into this file. See category_actions.js, for example.